README.md

```diff
@@ -1,92 +1,111 @@
-  - #### Docker
-    - [Docker](https://docs.docker.com/get-docker/)
+#### Terraform Requirements:
+- [Terraform](https://www.terraform.io/downloads.html) - v1.0.0
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl) - v1.19.9
+- [jq](https://stedolan.github.io/jq/) - v1.6
+- [Azure CLI](https://docs.microsoft.com/en-us/cli/azure) - (optional - useful as an alternative to the Azure Portal) - v2.24.2
 
+#### Docker Requirements:
+- [Docker](https://docs.docker.com/get-docker/)
 
 ## Getting Started
 
-### Clone this project
+When you have prepared your environment with the prerequisites, you are ready to obtain and customize the Terraform scripts that
+will set up your Kubernetes cluster.
 
-Run these commands in a Terminal session:
+### Clone this Project
+
+Run the following commands from a terminal session:
 
 ```bash
 # clone this repo
 git clone https://github.com/sassoftware/viya4-iac-azure
 
-# move to directory
+# move to the project directory
 cd viya4-iac-azure
 ```
 
-### Authenticating Terraform to access Azure
+### Authenticating Terraform to Access Microsoft Azure
+
+The Terraform process manages Microsoft Azure resources on your behalf. In order to do so, it needs your Azure account information and a user
+identity with the required permissions. See [Terraform Azure Authentication](./docs/user/TerraformAzureAuthentication.md) for details.
 
-See [Terraform Azure Authentication](./docs/user/TerraformAzureAuthentication.md) for details.
+### Customizing Input Values
 
-### Customize Input Values
+Terraform scripts require variable definitions as input. Many variables do not have default values. Create a file named
+`terraform.tfvars` to customize any input variable value documented in the [CONFIG-VARS.md](docs/CONFIG-VARS.md) file. 
 
-Create a file named `terraform.tfvars` to customize any input variable value documented in the [CONFIG-VARS.md](docs/CONFIG-VARS.md) file. For starters, you can copy one of the provided example variable definition files in `./examples` folder. For more details on the variables declared refer to the [CONFIG-VARS.md](docs/CONFIG-VARS.md) file.
+To get started, you can copy one of the example variable definition files provided in the `./examples` folder. For more information about the
+variables that are declared in each file, refer to the [CONFIG-VARS.md](docs/CONFIG-VARS.md) file.
 
-When using a variable definition file other than `terraform.tfvars`, see [Advanced Terraform Usage](docs/user/AdvancedTerraformUsage.md) for additional command options.
+You have the option to specify variable definitions that are not included in `terraform.tfvars` or to use a variable definition file other than
+`terraform.tfvars`. See [Advanced Terraform Usage](docs/user/AdvancedTerraformUsage.md) for more information.
 
 ## Creating and Managing the Cloud Resources
 
-Create and manage the cloud resources by either: 
+Create and manage the required cloud resources. Take one of the following steps: 
```

Comment:
Take one of the following steps -> Choose one of the following steps.

As a note we do have a preference to steer the client toward using Docker. Not sure if that's mentioned here, but it should be the default when referencing items.